From c47007aa04b37e600c8c21620209cd038ca6b42a Mon Sep 17 00:00:00 2001
From: Paul Menzel <paulepanter@users.sourceforge.net>
Date: Wed, 10 Oct 2012 11:09:59 +0200
Subject: [PATCH] burn-libburn-common.c: Use std:/tmp/my_emulated_drive

A medium has to be put into the burner though, so that Brasero allows you to press the burn button.
---
 plugins/libburnia/burn-libburn-common.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/libburnia/burn-libburn-common.c b/plugins/libburnia/burn-libburn-common.c
index 1965b86..11364bd 100644
--- a/plugins/libburnia/burn-libburn-common.c
+++ b/plugins/libburnia/burn-libburn-common.c
@@ -164,7 +164,7 @@ brasero_libburn_common_ctx_new (BraseroJob *job,
 
 	/* we just want to scan the drive proposed by drive */
 	brasero_job_get_device (job, &device);
-	res = burn_drive_convert_fs_adr (device, libburn_device);
+	res = burn_drive_convert_fs_adr ("stdio:/tmp/my_emulated_drive", libburn_device);
 	g_free (device);
 	if (res <= 0) {
 		g_set_error (error,
-- 
1.7.10.4

